packages/sessions-sdk-react/src/session-provider.tsx

```diff
@@ -474,7 +474,6 @@ const getNextState = (
       }
       case StateType.CheckingStoredSession:
       case StateType.Established:
-      case StateType.RestoringSession:
```

Comment:
This state had been replaced by `CheckingStoredSession` and I guess I forgot to remove the dead code